xonsh/xontribs.json

```diff
@@ -80,6 +80,11 @@
   "package": "xontrib-z",
   "url": "https://github.com/astronouth7303/xontrib-z",
   "description": ["Tracks your most used directories, based on 'frecency'."]
+ },
+ {"name": "autojump",
+  "package": "xonsh",
+  "url": "http://xon.sh",
+  "description": ["Add autojump support into xonsh"]
```

Comment:
This should probably contain a little more documentation on how to actually use the xontrib (`j` command, environment variables you added, etc). This serves as the only user facing documentation the xontrib has.